app/views/favourites/favourite_quotes.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @user_favourites.each do |quote| %>
+  <div class="well">
+    <%=  render "quotes/quote", quote: quote %>
```

Comment:
Yeah `render` will try and figure out what you're trying to render from the `object` you're passing it. But if the partial was in a /shared folder or something, then you'd need to specify.